ajax/libs/mdbootstrap/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "mdbootstrap",
+  "filename": "js/mdb.min.js",
+  "version": "4.1.1",
+  "description": "Material Design for Bootstrap (MDB) is a powerful Material Design UI KIT for most popular HTML, CSS, and JS framework - Bootstrap.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mdbootstrap/bootstrap-material-design.git"
+  },
+  "keywords": [
+    "material",
+    "design",
+    "bootstrap",
+    "mdbootstrap",
+    "mdb",
+    "framework",
+    "css",
+    "mobile-first",
+    "responsive"
+  ],
+  "author": "MDBootstrap",
+  "license": "MIT",
+  "homepage": "https://mdbootstrap.com/material-design-for-bootstrap/",
+  "npmName": "mdbootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "+(js|css)/mdb*",
```

Comment:
Thanks @PeterDaveHello ,
I want to ask something.
How did you determine that we need `**` under `css|js` folder in this case?
Thanks!